### ISSUE TITLE
Superposition de deux observations

### ISSUE THREAD

**Issue by TheoLechemia** [PnX-SI/GeoNature-atlas#290]
Le mode cluster est désactivé au deçà d'un nombre d'observation. Du coup certains observation ayant la même geoloc sont invisibles.

**Comment by camillemonchicourt** [comment]
Oui, comme on avait évoqué il serait certainement mieux d'avoir un comportement unique pour toutes les fiches, et de permettre à l'utilisateur de switcher lui-même entre un mode point et un mode cluster.
